### PR TITLE
Add a link to the QuTiP Virtual Lab.

### PR DESCRIPTION
**Pull request by hodgestar**
An initial pass at adding QuTiP Virtual Lab to the website.

The plan is to use the blurb here as a page to link to when announcing QVL elsewhere.

Things left todo:

- [ ] @trentfridey @ajgpitch reviews of the blurb text would be particularly welcome.
- [ ] Check whether this even renders in whatever version of Bootstrap is used by the renderer.
- [ ] Add a news item.

Probably the QuTiP website needs a bigger revamp, but I'm declaring that out of scope of this PR. :)

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
         <li><a href="/documentation.html">Users Guide</a></li>
         <li><a href="/qutip-tutorials/">Tutorials</a></li>
         <li><a href="/qutip-benchmark/">Benchmarks</a></li>
+        <li><a href="/qutip-virtual-lab.html">QuTiP Virtual Lab</a></li>
```
Comment (trentfridey) [inline _includes/navbar.html:23]: I think the `.html` will break this link
```suggestion
        <li><a href="/qutip-virtual-lab">QuTiP Virtual Lab</a></li>
```

File: images/qutip-virtual-lab.jpg
Comment (trentfridey) [inline images/qutip-virtual-lab.jpg:1]: I made a small update to the logo if you want to use this:
share

**Review by trentfridey — COMMENTED** [review]
You have my approval on the text blurb

File: _includes/navbar.html (hunk 1 above)
Comment (hodgestar) [reply _includes/navbar.html:23]: This is a link to the HTML file with the blurb.

File: images/qutip-virtual-lab.jpg
Comment (hodgestar) [reply images/qutip-virtual-lab.jpg:1]: Update applied. Also nice to have a PNG instead of a JPEG.

**Comment by hodgestar** [comment]
Thanks for reviewing @trentfridey. Sorry this took a long time to get back to.

**Pull request merged**